src/registrar/fixtures.py

```diff
@@ -101,6 +106,11 @@ class UserFixture:
             "first_name": "Rebecca-Analyst",
             "last_name": "Hsieh-Analyst",
         },
+        {
+            "username": "5dc6c9a6-61d9-42b4-ba54-4beff28bac3c",
+            "first_name": "David-Analyst",
+            "last_name": "Kennedy-Analyst"
```

Comment:
Same as above, on line 112  it should be `"last_name": "Kennedy-Analyst",` and that'll resolve the linting issue I think :-)